### DISCUSSION TITLE
Multiple Domains on a given stage?

### DISCUSSION THREAD

**Discussion by amerine** [kubero-dev/kubero#352] [Q&A] [answered]
Hey hey! I'd love to know if it's possible (and how to if so) to add multiple domains to a given app in a stage? It appears that an app currently only supports a single domain, is that true?

**Comment by mms-gianni** [comment, marked as answer]
Hi @amerine 

This is on my Bucketlist and will be added soon.

**Reply by amerine** [reply]
Is there anything one could do to help out? Are PRs welcome or you want to do this on your own?

**Reply by mms-gianni** [reply]
Sure. Help and PR's are very welcome. 

I've opened an feature request issue on that here to keep all the information in one place: https://github.com/kubero-dev/kubero/issues/363
